File: PYTHON/Program/DetailElektronik.py
from KategoriProduk import KategoriProduk

class DetailElektronik(KategoriProduk):
    def __init__(self, id_produk, nama_produk, harga, stok,
                 id_kategori, nama_kategori,
                 id_elektronik, garansi, daya, spesifikasi):
        super().__init__(id_produk, nama_produk, harga, stok, id_kategori, nama_kategori)
        self.id_elektronik = id_elektronik
        self.garansi = garansi
        self.daya = daya
        self.spesifikasi = spesifikasi

    def tampilkan_info(self):
        super().tampilkan_info()
        print(f"ID Elektronik : {self.id_elektronik}")
        print(f"Garansi       : {self.garansi}")
        print(f"Daya          : {self.daya}")
        print(f"Spesifikasi   : {self.spesifikasi}")

File: JAVA/Program/DetailElektronik.java
public class DetailElektronik extends KategoriProduk {
    private String idElektronik, garansi, daya, spesifikasi;

    public DetailElektronik(String idProduk, String namaProduk, int harga, int stok,
                            String idKategori, String namaKategori,
                            String idElektronik, String garansi, String daya, String spesifikasi) {
        super(idProduk, namaProduk, harga, stok, idKategori, namaKategori);
        this.idElektronik = idElektronik;
        this.garansi = garansi;
        this.daya = daya;
        this.spesifikasi = spesifikasi;
    }

    @Override
    public void tampilkanInfo() {
        super.tampilkanInfo();
        System.out.println("ID Elektronik : " + idElektronik);
        System.out.println("Garansi       : " + garansi);
        System.out.println("Daya          : " + daya);
        System.out.println("Spesifikasi   : " + spesifikasi);
    }
}
